<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 255, 211);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1121</width>
      <height>921</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 18pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>题库名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="title">
         <property name="styleSheet">
          <string notr="true">font: 75 18pt &quot;Adobe Devanagari&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectbutton">
         <property name="styleSheet">
          <string notr="true">font: 75 18pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>选择题库</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox1">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;Adobe Devanagari&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox2">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;Adobe Devanagari&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox3">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;Adobe Devanagari&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="question">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="xuanxiang">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBoxA">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxB">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxC">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxD">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasttimu">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>上一题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="submition">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showresult">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>展示答案</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nexttimu">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Adobe Devanagari&quot;;</string>
         </property>
         <property name="text">
          <string>下一题</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="jiexi">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Adobe Devanagari&quot;;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Adobe Devanagari'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
